src/wms/src/test/java/org/geoserver/wms/WMSTest.java

```diff
@@ -44,6 +50,16 @@
     @Override
     protected void onSetUp(SystemTestData testData) throws Exception {
         super.onSetUp(testData);
+        Catalog catalog = getCatalog();
+        testData.addStyle("burg-fill", "./map/burg-fill.sld", WMSTest.class, catalog);
+
+        // copy over the svg icon
+        File styles = new File(testData.getDataDirectoryRoot(), "styles");
+        File burg = new File(styles, "burg02.svg");
+        try(InputStream is = getClass().getResourceAsStream("./map/burg02.svg"); FileOutputStream fos = new FileOutputStream(burg)) {
+            IOUtils.copy(is, fos);
+        }
+
```

Comment:
This doesn't seem to be referenced anywhere in the test case that was added. Why is this here?